### ISSUE TITLE
The Border Color for the Optional E-mail Address Field Does Not have Enough Color Contrast

### ISSUE THREAD

**Issue by jfc3-dol** [mdn/content#26860]
### MDN URL

https://developer.mozilla.org/en-US/docs/Learn/Forms/UI_pseudo-classes

### What specific section or headline is this issue about?

Styling inputs based on whether they are required or not

### What information was incorrect, unhelpful, or incomplete?

The border color for the optional e-mail address field ( Email address (include if you want a response): ) in the feedback form does not have enough color contrast. The color contrast ratio is 1.8:1 for the border of #C0C0C0 on a white background.

Plus, it looks like it is disabled INPUT field.

### What did you expect to see?

I expected the border to have enough contrast to see it.

### Do you have any supporting links, references, or citations?

nope

### Do you have anything more you want to share?

nope

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/learn/forms/ui_pseudo-classes`
* MDN URL: https://developer.mozilla.org/en-US/docs/Learn/Forms/UI_pseudo-classes
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/learn/forms/ui_pseudo-classes/index.md
* Last commit: https://github.com/mdn/content/commit/65b9418c7d0e3a331ac50249adf0024f44789923
* Document last modified: 2023-05-09T00:34:38.000Z

</details>

**Comment by bsmth** [comment]
Hi @jfc3-dol - thanks for raising this. Shall we just improve it to an obvious degree like:

```css
input:optional {
  border: 3px solid rebeccapurple;
}
```

See https://developer.mozilla.org/en-US/docs/Web/CSS/:optional#examples

**Comment by jfc3-dol** [comment]
@bsmth 

You're welcome.

To not complicate things and not make them different I would make the INPUT field border for the e-mail address match the first and last name INPUT field borders.

**Comment by queengooborg** [comment]
> To not complicate things and not make them different I would make the INPUT field border for the e-mail address match the first and last name INPUT field borders.

Using different colors is the whole point of this section: "Styling inputs based on whether they are required or not".  This example is meant to show how CSS can be applied to required vs. optional, so setting both required and optional fields to the same color defeats the purpose of the example.

> Shall we just improve it to an obvious degree

I'd also recommend increasing the border size to 2px, as there doesn't seem to be enough contrast between black and `rebeccapurple` (or really, any other color that would retain a good contrast with the white background).
